community/cypher/cypher-compiler-3.1/src/main/scala/org/neo4j/cypher/internal/compiler/v3_1/planner/logical/patternExpressionRewriter.scala

```diff
@@ -63,8 +67,32 @@ case class patternExpressionRewriter(planArguments: Set[IdName], context: Logica
             val arguments = planArguments ++ scopeMap(expr)
             val (plan, namedExpr) = context.strategy.planPatternExpression(arguments, expr)(context)
             val uniqueNamedExpr = namedExpr.copy()
-            val nestedPlan = NestedPlanExpression(plan, uniqueNamedExpr)(uniqueNamedExpr.position)
-            acc.updated(expr, nestedPlan)
+            val path = ast.EveryPath(namedExpr.pattern.element)
+            val step: PathStep = projectNamedPaths.patternPartPathExpression(path)
+            val pathExpression: PathExpression = ast.PathExpression(step)(expr.position)
+
+            val rewrittenExpression = NestedPlanExpression(plan, pathExpression)(uniqueNamedExpr.position)
+            acc.updated(expr, rewrittenExpression)
+          }
+
+          (newAcc, Some(identity))
+
+      // replace pattern comprehension
+      // the contained pattern expression for no further processing
+      // by this tree fold
```

Comment:
I don't understand this comment. I see it was only copy-pasted from above. Could it be improved or removed? What does it mean?